apps/cli/src/node/driver/k8s.ts

```diff
@@ -0,0 +1,64 @@
+import { execa } from "execa";
+import semver from "semver";
+
+import { NodeDriver, NodeStatus } from "./index.js";
+import { DApp } from "../database/index.js";
+
+export class K8sDriver implements NodeDriver {
+    constructor() {}
+
+    async init(): Promise<void> {
+        // check helm installed version
+        const { stdout: helmVersion } = await execa("helm", [
+            "version",
+            "--template",
+            "{{.Version}}",
+        ]);
+
+        const minimumVersion = ">=3.0.0";
+        if (!semver.satisfies(helmVersion, minimumVersion)) {
+            throw new Error(`Helm version must be ${minimumVersion}`);
+        }
+    }
+
+    async start(dapp: DApp, location: string): Promise<NodeStatus> {
+        const name = dapp.address.substring(2, 8).toLocaleLowerCase();
+        const chart = "oci://docker.io/cartesi/rollups-validator-node";
+        const httpProvider = "http://host.docker.internal:8545";
+        const wsProvider = "ws://host.docker.internal:8545";
+        const serverManager = "cartesi/server-manager:0.9.1";
```

Comment:
```suggestion
        const serverManager = "cartesi/server-manager:0.7.0";
```

Reply:
this won't work, since the current rollups-validator-node helm charts requires a DApp image that's based on the server-manager and embeds the machine-snapshot.

I shall work on a fixup for that to be optional.